doc/manuals/user/v1_v2_coexistence.md

```diff
@@ -148,3 +149,9 @@ practical advantage of having more than one, from a functional point of view, e.
 [pending on a more definitive solution](https://github.com/telefonicaid/fiware-orion/issues/3044).
 
 [Top](#top)
+
+## Context availability subscriptions
+
+Note that context availability subscriptions and notifications are not included in NGSIv2. They have been intentionally left out due to it is a feature in NGSIv1 that is rarely used and doesn't worth the effort of being included in NGSIv2.
```

Comment:
... and it isn't worth the effort to include it in NGSIv2

Reply:
Fixed in bb00131